draft-ietf-quic-multipath.md

```diff
@@ -372,58 +372,45 @@ Reset ({{Section 10.3 of QUIC-TRANSPORT}}) closes the connection.
 Both endpoints, namely the client and the server, can initiate path closure,
 by sending a PATH_ABANDON frame (see {{path-abandon-frame}}) which
 requests the peer to stop sending packets with the corresponding Destination Connection ID.
-The PATH_ABANDON frame contains the Destination Connection
-ID Sequence Number and therefore can be sent on any path.
-
-Once a path is
-marked as "abandoned", it means that the resources related to the path,
-such as the used connection IDs, can be released.
-However, information related to data delivered over that path SHOULD
-not be released immediately as acknowledgments can still be received
-or other frames that also may trigger retransmission of data on another
-path.
-
-The endpoint sending the PATH_ABANDON frame SHOULD consider a path as
-abandoned when the packet that contained the PATH_ABANDON frame is
-acknowledged. When releasing resources of a path, the endpoint SHOULD
-send a RETIRE_CONNECTION_ID frame for the connection IDs used on the path,
-if any.
 
-The receiver of a PATH_ABANDON frame SHOULD NOT release its resources
-immediately, but SHOULD wait for the reception of the RETIRE_CONNECTION_ID
-frame for the used connection IDs or 3 RTOs.
+The receiver of a PATH_ABANDON frame should not release its resources
+immediately, but SHOULD wait for at least three times the current
```

Comment:
~~~suggestion
The sender and the receiver of a PATH_ABANDON frame should not release their resources
~~~